sample/ReactNativeBasicPlayerSample/App.js

```diff
@@ -293,9 +258,10 @@ export default class App extends Component {
     if (this.state.currentTracker === undefined) {
       console.log('tracker is null, cannnot track seek');
       return;
-    }
+    
     this.state.currentTracker.trackEvent(ACPMediaEvent.EventSeekStart);
   }
```

Comment:
`this.state.currentTracker.trackEvent(ACPMediaEvent.EventSeekStart);`  should be after the `if` block.

Reply:
updated